console.log("Iniciando el script...");

(async () => {
    try {
        console.log("Esto es un mensaje de prueba");
    } catch (error) {
        console.error("Error encontrado:", error);
    }
})();